userspace/uv/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "agnos"
+version = "0.1.0"
+authors = [{name = "Vehicle Researcher", email = "user@comma.ai"}]
+requires-python = "~=3.11"
+description = ""
+dependencies = [
+  "openpilot",
```

Comment:
why can't we use the git url?

Reply:
uv has a bug:

- if I put "openpilot@git+https://github.com/commaai/openpilot" in dependencies it will crash trying to pull submodules. I saved a [log](https://gist.github.com/andiradulescu/6b94eae07bd297e74668d42a53b85d08).
- works fine with pip install, so it's something related to uv.

That's why I decided to curl master pyproject.toml. Also, this is much faster, since cloning the whole openpilot repo takes precious time.